shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -306,13 +309,14 @@
 
   int currentClippingCount = CountClips(params.mutatorsStack);
   int previousClippingCount = clip_count_[view_id];
+
   if (currentClippingCount != previousClippingCount) {
     clip_count_[view_id] = currentClippingCount;
     // If we have a different clipping count in this frame, we need to reconstruct the
     // ClippingChildView chain to prepare for `ApplyMutators`.
     UIView* oldPlatformViewRoot = root_views_[view_id].get();
-    UIView* newPlatformViewRoot =
-        ReconstructClipViewsChain(currentClippingCount, touchInterceptor, oldPlatformViewRoot);
+    UIView* newPlatformViewRoot = ReconstructClipViewsChain(currentClippingCount, touchInterceptor,
+                                                            oldPlatformViewRoot, view_id);
```

Comment:
nit: feels like view_id is enough information to get the touchInterceptor and the oldPlatformViewRoot, may be cleaner to either just pass view_id, or alternatively to pass the view's position.